pyproject.toml

```
@@ -68,4 +68,4 @@ force_grid_wrap = 0
 line_length = 88
 indent = "    "
 # will group `import x` and `from x import` of the same module.
-force_sort_within_sections = true
+force_sort_within_sections = true
```

```suggestion
force_sort_within_sections = true

```

We always end with an empty line.